cgmanifests/README.md

```diff
@@ -0,0 +1,14 @@
+This directory contains Component Governance cgmanifest.json files.
+See here for more details: https://docs.opensource.microsoft.com/tools/cg/cgmanifest.html
+
+Any cgmanifest file entries that don't belong in more specific categories
+(e.g., git submodules) can go into `cgmanifests/cgmanifest.json`.
+
+### Git Submodules
```

Comment:
Not usual to have `###` and not `#` and `##` before, as the headings are hierarchical